M2/Macaulay2/packages/GKMManifolds/Documentation_GKMManifolds.m2

```diff
@@ -0,0 +1,2165 @@
+			toric variety is smooth (or simplicial) it is naturally a GKM manifold:
+			Let $X$ be a smooth toric variety 
+			and $U$ be an affine chart whose associated character lattice is generated by elements of
+			weights $a_1,\dots, a_m$. Then $(\mathbb C^*)^n$ acts on $U$ by 
+			$t \cdot (x_1,\dots, x_n) = (t^{-a_1}x_1,\dots, t^{-a_n}x_n)$.
+			We caution that this package uses the outer normals instead of inner normals.
+			
+		Text	
+			The method @TO "normalToricVariety"@ from the package @TO "NormalToricVarieties"@ allows the 
+			user to construct smooth toric varieties. To convert it to a GKM manifold we use the method
+			@TO "tVariety"@. Here is an example with $X = Bl_p\mathbb P^2$, the blow-up of $\mathbb P^2$ at a point,
+			which is also the first Hirzebruch surface.
+		Example
+			FF1 = hirzebruchSurface 1;
+			X = tVariety FF1;
+			peek FF1
+			peek X
+
+   		Text
+			If a $T$-variety $X$ was originally constructed from @TO "normalToricVariety"@ we can convert it
+			back to a toric variety.
+		Example
+			Y = normalToricVariety(X); -- X defined in the previous example above
+			Y === FF1
+		Text
+			Continuing this example, the following shows how to convert a $T$-invariant divisor constructed using
+			@TO ToricDivisor@ to a @TO TKClass@.
+
+		Example
+			antiK = - toricDivisor(FF1) -- the anti-canonical class on FF1
+			TantiK = tKClass(X,antiK)
+			isWellDefined TantiK
+
+		Text
+			Since the toric variety $X = Bl_p\mathbb P^2$ is Gorenstein Fano, 
+			with its anticanonical embedding in $\mathbb P^8$, the T-equivariant Euler characteristic of
+			the anticanonical divisor is the sum of the characters of the sections of the associated line bundle.
+
+		Example
+			tChi TantiK
+
+		Text
+			We caution the following difference in convention:
+			Projective $n$-space $\mathbb P^n$ as a @TO NormalToricVariety@ constructed using @TO "toricProjectiveSpace"@
+			is acted upon by an $n$-dimensional torus. However, as a @TO TVariety@
+			constructed using @TO "tProjectiveSpace"@, it is acted upon by an $(n+1)$-dimensional torus.
+		Example
+			X = tVariety toricProjectiveSpace 2; -- the torus is C^2
+	   		Y = tProjectiveSpace 2; -- the torus is C^3
+			peek X
+			peek Y		
+
+	SeeAlso
+		(tVariety, NormalToricVariety)
+		tProjectiveSpace
+		normalToricVariety
+		(tKClass, TVariety, ToricDivisor)
+
+///
+
+
+doc ///
+	Key
+		(normalToricVariety, TVariety)
+	Headline
+		converts a T-variety back into a toric variety
+	Usage
+		Y = normalToricVariety X
+	Inputs
+		X:TVariety
+	Outputs
+		Y:NormalToricVariety		
+	Description			
+		Text	
+			If $X$ is a $T$-variety that was originally constructed using @TO "normalToricVariety"@, then
+			this method reverts $X$ to a @TO "NormalToricVariety"@.
+			
+		Example
+			X = toricProjectiveSpace 2;
+			Y = tVariety X
+			assert(normalToricVariety Y === X)
+
+	SeeAlso
+		tVariety
+		normalToricVariety
+
+///
+
+
+
+
+doc ///
+	Key
+		TVariety
```

Comment:
"T" in "TVariety" doesn't mean torus - it's just a commonly used single letter variable.  What about "TorusVariety" instead?